pkg/inframetadata/internal/hostmap/constants.go

```diff
@@ -0,0 +1,36 @@
+package hostmap
+
+import (
+	conventions "go.opentelemetry.io/collector/semconv/v1.18.0"
+)
+
+// Custom attributes not in the OpenTelemetry specification
+const (
+	attributeKernelName    = "os.kernel.name"
+	attributeKernelRelease = "os.kernel.release"
+	attributeKernelVersion = "os.kernel.version"
+)
+
+// gohai platform subsection fields.
+const (
+	platformHostname         = "hostname"
```

Comment:
To be consistent with the above, I'd prefix these with `field` too.

Reply:
Done